testing/ct/Dockerfile

```diff
@@ -0,0 +1,17 @@
+#TODO: build/tag/use stable hms-test:3.1.0 image after PR is merged
+#FROM artifactory.algol60.net/csm-docker/stable/hms-test:3.1.0
+FROM artifactory.algol60.net/csm-docker/unstable/hms-test:3.1.0-20220331211520.329a922
```

Comment:
Lets pin this back on stable/hms-test:3.0.0

Reply:
Done.